sdk/core/package.json

```diff
@@ -63,14 +63,14 @@
     "keccak256": "^1.0.0",
     "lodash.keyby": "^4.6.0",
     "lodash.uniq": "^4.5.0",
-    "node-fetch": "^2.6.0",
     "randombytes": "^2.1.0",
     "secp256k1": "^4.0.2",
     "tslib": "^2.0.0",
     "uuid": "^8.3.1"
   },
   "devDependencies": {
     "@affinidi/eslint-config": "1.0.1",
+    "@affinidi/platform-fetch-node": "^1.0.0",
```

Comment:
It's dev dependency because it's only used in tests?

Reply:
I think yes

Reply:
yes, it is only used in tests (which are run on node)